build.gradle

```diff
@@ -37,36 +42,24 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
-/*configure(subprojects.findAll { !it.name.contains("testpack") && !it.name.startsWith("module") }) {
+configure(subprojects.findAll { !it.name.contains("testpack") && !it.name.startsWith("module") && !it.name.contains("android")}) {
+    // General Java configuration
+    apply plugin: 'java'
```

Comment:
This plugin isn't really appropriate for any of the projects (they should all be android or java-library)

Reply:
I might be able to get around that by restructuring the Artifactory setup some more. Our usual standard relies on a few of the tasks in the plain java plugin that the java-library doesn't have. But really that's more just me having yet to dig in deeper on how the java-library works, will do 👍